jekyll/_cci2/writing-yaml.md

```diff
@@ -0,0 +1,171 @@
+---
+layout: classic-docs
+title: "Writing YAML"
+short-title: "Writing YAML"
+description: "How to Write YAML on CircleCI"
+categories: [configuring-jobs]
+order: 20
+---
+
+*[Basics]({{ site.baseurl }}/2.0/basics/) > Writing YAML*
+
+This document describes the most important features of YAML
+for use in CircleCI configuration.
+
+* TOC
+{:toc}
+
+## Overview
+
+[YAML](http://yaml.org) is a human-friendly data serialization standard for all programming languages.
+It is a strict superset of [JSON](https://www.json.org/),
+another data serialization language.
+This means it can do everything JSON can... and more.
+
+CircleCI configuration is stored in a single YAML file located at `~/.circleci/config.yml`,
+where `~` is the root of your project's directory.
+Since most of your work with CircleCI occurs in this file,
+it is important to understand the basics of YAML formatting.
+
+## How to Write YAML
+
+The basic structure of a YAML file is a [map](https://en.wikipedia.org/wiki/Map_(higher-order_function))
```

Comment:
That link should refer to a hashmap article: https://en.wikipedia.org/wiki/Hash_table